Google.Api.Gax.Grpc/GrpcAdapter.cs

```diff
@@ -84,5 +104,29 @@ private static GrpcAdapter DetectDefaultPreferringGrpcNetClient()
                 return GrpcCoreAdapter.Instance;
             }
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="grpcApiDescriptor"></param>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        public static GrpcAdapter GetFallbackAdapter(GrpcApiDescriptor grpcApiDescriptor)
+        {
+            // TODO: Some way of indicating a preference? Or just set the adapter in the client builder...?
```

Comment:
I think this is fine. For a specific adapter they have the builder property and DI. For the fallback is fine if we get to decide.

Reply:
Yup. I suspect it's fine too. Let's leave this TODO here for now just in case.